src/controller/scan/callback.go

```diff
@@ -92,7 +92,7 @@ func scanTaskStatusChange(ctx context.Context, taskID int64, status string) (err
 
 		robotID := getRobotID(t.ExtraAttrs)
 		if robotID > 0 {
-			if err := robotCtl.Delete(ctx, robotID); err != nil {
+			if err := robotCtl.Delete(ctx, robotID, &robot.Option{Operator: "job-service"}); err != nil {
```

Comment:
should we use `harbor-jobservice` to keep uniformity? https://github.com/goharbor/harbor/blob/969384cd6340f67ceb2d811309c23ec0b1e343ad/src/common/secret/store.go#L19-L20

Reply:
updated